.github/workflows/zulip_emoji_merge_delegate.yaml

```diff
@@ -0,0 +1,32 @@
+name: Zulip Emoji Merge Delegate
+
+on:
+  schedule:
+    - cron: '0 * * * *'  # Runs every hour
+
+jobs:
+  zulip-emoji-merge-delegate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout mathlib repository
+      uses: actions/checkout@v4
```

Comment:
I don't understand a large part of this script, but does it actually use the repository, or does it just access mathlib information online?

Reply:
Thinking about this again, the script itself should be checked out!

Maybe you can do add a `sparse-checkout` flag to the action and just download the script itself, if that is really all that is needed by the action.